### PR TITLE
first commit of view-course-details

### PR DESCRIPTION
**Pull request by poojithaharinath**
Please review my code

### CODE REVIEW

File: .project [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Murutechi) [inline .project:1]: Remove .project file from feature branch

File: src/main/java/com/jm/lms/studentms/controller/CourseDetailsController.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.jm.lms.studentms.model.CourseDetails;
+import com.jm.lms.studentms.service.CourseDetailsService;
+
+@RestController
+@RequestMapping("api/v1/lms/coursedetails")
+public class CourseDetailsController {
```
Comment (Murutechi) [inline src/main/java/com/jm/lms/studentms/controller/CourseDetailsController.java:24]: Change class name to ViewCourseDetailsController

File: src/main/java/com/jm/lms/studentms/repository/CourseDetailsRepository.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package com.jm.lms.studentms.repository;
```
Comment (Murutechi) [inline src/main/java/com/jm/lms/studentms/repository/CourseDetailsRepository.java:1]: Change repository name to ViewCourseDetailsRepository

File: src/main/java/com/jm/lms/studentms/service/CourseDetailsService.java [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.util.ReflectionUtils;
+
+import com.jm.lms.studentms.model.CourseDetails;
+import com.jm.lms.studentms.repository.CourseDetailsRepository;
+
+@Service
+public class CourseDetailsService {
```
Comment (Murutechi) [inline src/main/java/com/jm/lms/studentms/service/CourseDetailsService.java:16]: Change service name to ViewCourseDetailsService

File: target/classes/META-INF/MANIFEST.MF [hunk 5]
```diff
@@ -0,0 +1,6 @@
+Manifest-Version: 1.0
```
Comment (Murutechi) [inline target/classes/META-INF/MANIFEST.MF:1]: Remove target folder from feature branch

**Review by Murutechi — CHANGES_REQUESTED** [review]

**Comment by poojithaharinath** [comment]
I have edited the changes you requested, you can please check it.

**Comment by poojithaharinath** [comment]
I have edited the changes you requested, please check it.

**Pull request closed without merging**